Header header

int32 strength
int32 frequency

